cluster/gce/gci/configure-helper.sh

```diff
@@ -53,6 +53,9 @@ function main() {
   CONTAINERIZED_MOUNTER_HOME="${KUBE_HOME}/containerized_mounter"
   PV_RECYCLER_OVERRIDE_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/pv-recycler-template.yaml"
   DEFAULT_NETWORK_TEMPLATE="${KUBE_HOME}/kube-manifests/kubernetes/default-network.tmpl"
+  DEFAULT_PATH="/hack/runtime"
+  DEFAULT_VPC_TEMPLATE_PATH="${DEFAULT_PATH}/default_mizar_network_vpc_template.json"
```

Comment:
is DEFAULT_PATH a new env var introduced here?

Reply:
This env is to match the path defined in golang code line 45 in cmd/kube-controller-manager/app/mizarcontrollers.go.

homeSubPath        = "/hack/runtime/"